README.md

```diff
@@ -107,60 +124,96 @@ iis_site { 'complete':
     },
   ],
   require => File['c:\\inetpub\\complete'],
-}
+} ->
 
 iis_virtual_directory { 'vdir':
   ensure       => 'present',
   sitename     => 'complete',
   physicalpath => 'c:\\inetpub\\complete_vdir',
-  require => File['c:\\inetpub\\complete_vdir'],
+  require      => File['c:\\inetpub\\complete_vdir'],
 }
 ```
 
 ## Reference
 
 ### Facts
 
-* `iis_version` - The version of the installed IIS. Empty string if not installed.
+* `iis_version` - The version of IIS that is installed. Empty string if not installed.
+
 
-### Types/Providers
+### Types and Providers
 
+* [iis_feature](#iis_feature)
 * [iis_application](#iis_application)
 * [iis_application_pool](#iis_application_pool)
-* [iis_feature](#iis_feature)
```

Comment:
Should probably move `iis_feature` back as this breaks the alphabetical ordering.